interface/main/finder/multi_patients_finder.php

```diff
@@ -0,0 +1,258 @@
+            margin-left: 10px;
+            margin-right: 10px;
+            display: inline-block;
+            vertical-align: middle;
+        }
+        .inline-box{
+            display: inline-block;
+            vertical-align: middle;
+        }
+        .remove-patient{
+            color: red;
+            pointer-events: auto;
+        }
+        #searchResultsHeader {
+            width: 100%;
+            border-collapse: collapse;
+        }
+        #searchResults {
+            width: 100%;
+            border-collapse: collapse;
+            background-color: white;
+            overflow: auto;
+        }
+
+        #searchResults .remove-patient {
+            cursor: hand;
+            cursor: pointer;
+        }
+        #searchResults td {
+            /*font-size: 0.7em;*/
+            border-bottom: 1px solid #eee;
+        }
+    </style>
+
+</head>
+
+<body class="body_top">
+<div class="container-responsive">
+    <div id="searchCriteria">
+        <form class="form-inline">
+            <div class="select-box">
+                <label><?php echo xlt('Patient name') .':'; ?></label>
+                <select id="by-name" class="input-sm">
+                    <option value=""><?php echo xlt('Type name'); ?></option>
+                </select>
+                <label><?php echo xlt('Patient ID'); ?></label>
+                <select id="by-id" class="input-sm">
+                    <option value=""><?php echo xlt('Type ID'); ?></option>
+                </select>
+            </div>
+            <div class="buttons-box">
+                <div class="inline-box">
+                    <button id="add-to-list"><?php echo xlt('Add to list'); ?></button>
+                </div>
+                <div class="inline-box">
+                    <button id="send-patients" onclick="selPatients()"><?php echo  xlt('OK'); ?></button>
+                </div>
+            </div>
+        </form>
+    </div>
+
+    <table id="results-table" class="table table-condensed">
+        <thead id="searchResultsHeader" class="head">
+        <tr>
+            <th class="srName"><?php echo  xlt('Name'); ?></th>
+            <th class="srPhone"><?php echo xlt('Phone'); ?></th>
+            <th class="srSS"><?php echo xlt('SS'); ?></th>
+            <th class="srDOB"><?php echo xlt('DOB'); ?></th>
+            <th class="srID"><?php echo xlt('ID'); ?></th>
+            <th></th>
+        </tr>
+        </thead>
+        <tbody id="searchResults">
+        <?php
+        if (isset($_GET['patients'])) {
+            foreach ($results as $index => $result) {
+                echo '<tr id="row'.$index.'">' .
+                        '<td>'. $result['lname'] . ', ' . $result['fname'] . '</td>' .
+                        '<td>' . $result['phone_home'] . '</td>' .
+                        '<td>' . $result['ss'] . '</td>' .
+                        '<td>' . $result['DOB'] . '</td>' .
+                        '<td>' . $result['pubpid'] . '</td>' .
```

Comment:
text() in 5 lines above

Reply:
I clean all the variables in the line 36. 
`$result=array_map('text', $result);`
This array is printed in the JS also, therefore I clean all the data in the php section.